Getting Started.

hwartig opened the pull request:
This PR contains my 'notes' that I took while setting up the project on my computer. It also adds a basic circle ci integration.

hwartig committed:
Add some more pretty common files to .gitignore

hwartig committed:
Add script/setup and requirements.txt

hwartig committed:
Add .circleci/config.yml

hwartig committed:
Add more helper scripts

hwartig committed:
add Getting started section to readme

exviva commented on scripts/test:
```diff
@@ -0,0 +1,8 @@
+
+# The unittest discover feature somehow doesn't pick up all tests
+# python -m unittest discover -s tests
+find tests -name 'test_*.py' -exec python3 -m unittest -v {} \;
```
We were actually using `nosetests` to run the tests, it does all the recursive file detection and path setting. Probably would make sense to add it to the readme somewhere :)

hwartig replied:
good to know I wasn't sure if I should add a testrunner to the requirements.txt but now I changed the runner to nose see 3687175

exviva reviewed (approved):
Sweet! ❤️

hwartig committed:
Use nosetests to run tests.

This is not only more convenient for console users but should also give us test result
reporting in circleci.
